taxcalc/tests/test_utils.py

```diff
@@ -138,9 +138,9 @@ def test_create_tables(cps_subsample):
                 0.07,
                 0.05,
                 0.02,
-                0.00,
+                np.nan,
                 np.nan]
```

Comment:
This needs to be reviewed further. @martinholmer @codykallen

Reply:
Why this change?

Reply:
Why this change?

Reply:
I suspect that it has something to do with [this example](https://pandas.pydata.org/pandas-docs/stable/basics.html#gotchas) from the Pandas docs.  However, I'm not sure why a why what should be a zero was converted to a NaN in our case.

Reply:
@codykallen you may be correct.  The result of     `print('sum', np.sum(calc1.records.c00100 > 10000000) )` is `0`.